git-keeper-client/gkeepclient/create_config.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2017 Thuy Dinh
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Provides the function for creating a new client.cfg file.
+"""
+
+from os.path import isfile, expanduser, isdir, join
+from gkeepclient.text_ui import confirmation
+from gkeepcore.system_commands import mkdir
+
+
+def create_config():
+    """
+    Prompts the user for configuration parameters and creates client.cfg
+    """
+    dirpath = expanduser("~/.config/git-keeper/")
+    filepath = join(dirpath, "client.cfg")
+    if not check_config_file(dirpath, filepath):
+        return
+    host_name = input("Host: ")
+    username = input("Username: ")
+    ssh_port = input("SSH port (press enter for port 22): ")
+    submissions_path = input("Submissions_path: ")
+    file_text = ("[server]\n" +
+                 "host = {}\n" +
+                 "username = {}\n").format(host_name, username)
+    if ssh_port != '':
+        file_text += ("# optional\n" +
+                      "ssh_port = {}\n\n").format(ssh_port)
+    if submissions_path != '':
+        file_text += ("# optional section\n" +
+                      "[local]\n" +
+                      "submissions_path = {}\n").format(submissions_path)
+    print("The following will be written to {}: ".format(filepath))
+    print(file_text)
+    if confirmation("Would you like to proceed?"):
+        with open(filepath, 'w') as config_file:
+            config_file.write(file_text)
+    else:
+        print("No file was written.")
+
+
+def check_config_file(dirpath, filepath):
+    """
+    Checks if client.cfg exist
```

Comment:
Use imperative voice (Check if ...). Also place a newline between the description and the parameters.